cdap-ui/app/cdap/services/helpers.js

```diff
@@ -295,6 +296,40 @@ const reverseArrayWithoutMutating = (array) => {
   return newArray;
 };
 
+const convertMapToKeyValuePairs = (map, addUniqueId = true) => {
+  if (addUniqueId) {
+    return Object.keys(map).map(mapKey => {
+      return {
+        key: mapKey,
+        value: map[mapKey],
+        uniqueId: 'id-' + uuid()
+      };
+    });
+  }
+  return Object.keys(map).map(mapKey => {
+    return {
+      key: mapKey,
+      value: map[mapKey],
+      uniqueId: 'id-' + uuid()
```

Comment:
should this not be here? Also we can start to use `Object.entries(map).map(([key, value]) => {...})` makes it simpler.